modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running multiple applications side-by-side
+* Clear boundaries for operations on what a Mule application is
+* Mule can monitor your applications and will reload configuration changes automatically
+* Applications can depend on different library versions, even if they would conflict before
```

Comment:
"even if they would conflict before" - what would conflict before, the applications, the library versions, or applications running difference library versions? Also, Library should be lowercase.

Reply:
I think it refers to "library versions". Could be changed to _Applications can depend on different library versions, even if these would conflict before_. or _Applications can depend on different library versions, even if the library versions would conflict before_ @aiannucci Could you reconfirm?

Reply:
Updated.